appinventor/appengine/src/com/google/appinventor/client/wizards/youngandroid/NewYoungAndroidProjectWizard.java

```diff
@@ -40,61 +63,53 @@
  *
  * @author markf@google.com (Mark Friedman)
  */
+
 public final class NewYoungAndroidProjectWizard {
+
   interface NewYoungAndroidProjectWizardUiBinder extends UiBinder<Dialog, NewYoungAndroidProjectWizard> {}
-  private static final NewYoungAndroidProjectWizard.NewYoungAndroidProjectWizardUiBinder UI_BINDER = GWT.create(NewYoungAndroidProjectWizard.NewYoungAndroidProjectWizardUiBinder.class);
+  private static final NewYoungAndroidProjectWizardUiBinder UI_BINDER = GWT.create(NewYoungAndroidProjectWizardUiBinder.class);
   private static final Logger LOG = Logger.getLogger(NewYoungAndroidProjectWizard.class.getName());
 
+  EditableProperty theme;
+  EditableProperty toolkit;
   // UI element for project name
   @UiField Dialog addDialog;
   @UiField Button addButton;
   @UiField Button cancelButton;
   @UiField LabeledTextBox projectNameTextBox;
-
+  @UiField YoungAndroidThemeChoicePropertyEditor themeEditor;
+  @UiField SubsetJSONPropertyEditor blockstoolkitEditor;
+  @UiField HorizontalPanel horizontalThemePanel;
+  @UiField HorizontalPanel horizontalBlocksPanel;
+  @UiField Label themeLabel;
+  @UiField Label blocksLabel;
+  String errorMessage = "";
```

Comment:
I'm not sure why this was made into a field when it is only accessed in the addProject method.